accelerator/config.h

```diff
@@ -67,6 +67,7 @@ typedef struct ta_config_s {
   char* mqtt_host;       /**< Address of MQTT broker host */
   char* mqtt_topic_root; /**< The topic root of MQTT topic */
 #endif
+  bool proxy_api_direct_pass; /**< Pass proxy api directly without processing */
```

Comment:
How about changing `proxy_api_direct_pass` to `proxy_api_passthrough`?

Reference: https://wiki.gentoo.org/wiki/GPU_passthrough_with_libvirt_qemu_kvm

Reply:
Alternatively, I think `proxy_passthrough` sounds good as well.

Reply:
sure